vizro-core/pyproject.toml

```diff
@@ -56,7 +56,7 @@ exclude_lines = [
   "if __name__ == .__main__.:",
   "if TYPE_CHECKING:"
 ]
-fail_under = 92
+fail_under = 91
```

Comment:
Oh no, what's happening?

Reply:
Since we don't call the `Vizro.build()` anymore (but `Vizro._pre_build()` instead), the `Vizro.build()` is recognised as not-tested method and all other methods called inside it (like `_build_action_loop_callbacks()`), hence the coverage is decreased.

Quick Question:
Code coverage is 92, so should we keep `fail_under` decremented by 1, so 91?

Reply:
I wouldn't worry about this too much - there is a ticket to properly test the `Vizro` class in this sprint so this will increase again. The `Vizro._prebuild` call should also be removed if possible

Reply:
Ok thtats ok, and more honest :)

Reply:
Thanks to both of you for reviewing the changes that should have been merged yesterday 💯.

@maxschulz-COL 
Quick Question:
Code coverage is 92, so should we keep fail_under decremented by 1, so 91?